test/services/overseas-entities/overseas.entities.mock.ts

```diff
@@ -47,6 +47,28 @@ export const ADDRESS: Address = {
     postcode: "BY 2"
 };
 
+export const PRIVATE_ADDRESS_MOCK = {
+    addressLine1: "addressLine1",
+    addressLine2: "addressLine2",
+    careOf: "care of information",
+    country: "country",
+    locality: "locality information",
+    poBox: "PO Box information",
+    postalCode: "postal code information",
+    premises: "premises information",
+    region: "region information"
+};
+
+export const MANAGING_OFFICERS_DATA_MOCK = {
+    managingOfficerAppointmentId: "123456789",
+    residentialAddress: ADDRESS_API,
+    principalAddress: ADDRESS_API,
```

Comment:
Assuming these need to be updated too?
```suggestion
    residentialAddress: PRIVATE_ADDRESS_MOCK,
    principalAddress: PRIVATE_ADDRESS_MOCK,
```